package.json

```
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node app.js",
+    "start": "node app.js NODE_ENV=production",
```

Можно лучше: продакшн режим не устанавливается при запуске `npm run start`.
Команда, которая указана в значении будет вызвана как системная команда. Например, в Linux она будет выполнена в bash. У команды определенный API:
```
[env] [command] [...args]
```
В данном случае `[env]` не передан, а остальные группируются так:
- `[command]` - `node`,
- `[...args]`:
  - `[arg0]` - `app.js`,
  - `[arg1]` - `NODE_ENV=production`

Надеюсь стало понятно:
```suggestion
    "start": "NODE_ENV=production node app.js",
```